getting-started/kubernetes/helm.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Installing Calico with Helm on Kubernetes
+description: Install Calico on a Kubernetes cluster using Helm 3.
+canonical_url: '/getting-started/kubernetes/helm'
+---
+
+### Big picture
+
+This installs Calico on a Kubernetes cluster using Helm 3.
+
+### Value
+
+Helm charts are a way to package up an application for Kubernetes.  This can be thought of as similar to the way `apt` or `yum` packages applications for operating systems.  
+
+Helm is also used by tools like ArgoCD to manage applications in a cluster, taking care of install, upgrade (and rollback if needed), etc. 
+
+### Before you begin
+
+- Install Helm 3.
+
+- Make sure you have a kubernetes cluster that meets the following requirements:
+  - kubernetes installed, but currently *without* a CNI plugin **OR** a cluster running a compatible CNI for Calico to run in policy-only mode 
+  - x86-64, arm64, ppc64le, or s390x processors
+  - RedHat Enterprise Linux 7.x+, CentOS 7.x+, Ubuntu 16.04+, or Debian 9.x+
+
+- Configure `kubeconfig` to work with your cluster (check by running `kubectl get nodes`)
+
+- Ensure that {{site.prodname}} can manage `cali` and `tunl` interfaces on the hosts.
+  If NetworkManager is present on the hosts, refer to
+  [Configure NetworkManager](../../maintenance/troubleshoot/troubleshooting#configure-networkmanager).
+
+### Concepts
+
+#### Operator based installation
+
+This guide uses Helm 3 to install the Tigera operator which will install {{site.prodname}}. The operator provides lifecycle management for Calico
+exposed via the Kubernetes API defined as a custom resource definition.
+
+### How to
+
+#### Download the helm chart
+
+1. Go to the Calico [releases page](https://github.com/projectcalico/calico/releases) and find the release you want to install.
+
+1. Download the chart from the release artifacts.  It will have a name like: `tigera-operator-{{page.version}}.0-1.tgz`
+
+#### Create a Helm Customization
+
+If you are installing on a cluster installed by EKS, GKE, AKS, Openshift or Docker Enterprise, you will need to customise this Helm chart.  
+
+You might also need to do this if you want to customise TLS certificates.
+
+Otherwise, this part can be skipped.
+
+1. Run `helm show values tigera-operator-{{page.version}}.0-1.tgz` to see the values that can be customised in the chart
+
+1. Create a `calico-config.yaml` file based on the output of `helm show values tigera-operator-{{page.version}}.0-1.tgz`.  
+
+1. If you are installing on a cluster installed by EKS, GKE, AKS, Openshift or Docker Enterprise, set the `kubernetesProvider` as described in the [Installation reference](../../reference/installation/api#operator.tigera.io/v1.Provider)
```

Comment:
I believe this is only required for Openshift thanks to auto-detection. But I could be wrong...

Reply:
We don't have tests to prove that autodetection works at this point.  Is it not safer to specify this value here?  We can remove it later once we're sure autodetection works.